spec/jobs/update_s3_inventory_spec.rb

```diff
@@ -9,8 +9,7 @@
     SiteSetting.enable_s3_inventory = true
 
     store = FileStore::S3Store.new
-    @client = Aws::S3::Client.new(stub_responses: true)
```

Comment:
There was no need to set `stub_responses: true` here. I removed it and the tests still pass.